smithy-kotlin-codegen/src/test/kotlin/software/amazon/smithy/kotlin/codegen/rendering/ClientConfigGeneratorTest.kt

```diff
@@ -53,65 +53,17 @@ class Config private constructor(builder: BuilderImpl): HttpClientConfig, Idempo
 """
         contents.shouldContainWithDiff(expectedProps)
 
-        val expectedJavaBuilderInterface = """
-    interface FluentBuilder {
-        fun endpointResolver(endpointResolver: EndpointResolver): FluentBuilder
-        fun httpClientEngine(httpClientEngine: HttpClientEngine): FluentBuilder
-        fun idempotencyTokenProvider(idempotencyTokenProvider: IdempotencyTokenProvider): FluentBuilder
-        fun sdkLogMode(sdkLogMode: SdkLogMode): FluentBuilder
-        fun build(): Config
-    }
-"""
-        contents.shouldContainOnlyOnceWithDiff(expectedJavaBuilderInterface)
-
-        val expectedDslBuilderInterface = """
-    interface DslBuilder {
-        /**
-         * Set the [aws.smithy.kotlin.runtime.http.operation.EndpointResolver] used to resolve service endpoints. Operation requests will be
-         * made against the endpoint returned by the resolver.
-         */
-        var endpointResolver: EndpointResolver?
-
-        /**
-         * Override the default HTTP client engine used to make SDK requests (e.g. configure proxy behavior, timeouts, concurrency, etc)
-         */
-        var httpClientEngine: HttpClientEngine?
-
-        /**
-         * Override the default idempotency token generator. SDK clients will generate tokens for members
-         * that represent idempotent tokens when not explicitly set by the caller using this generator.
-         */
-        var idempotencyTokenProvider: IdempotencyTokenProvider?
-
-        /**
-         * Configure events that will be logged. By default clients will not output
-         * raw requests or responses. Use this setting to opt-in to additional debug logging.
-         * This can be used to configure logging of requests, responses, retries, etc of SDK clients.
-         * **NOTE**: Logging of raw requests or responses may leak sensitive information! It may also have
-         * performance considerations when dumping the request/response body. This is primarily a tool for
-         * debug purposes.
-         */
-        var sdkLogMode: SdkLogMode
+        val expectedBuilder = """
+    class Builder {
+        var endpointResolver: EndpointResolver? = null
```

Comment:
fix

This is missing documentation on the generated builder properties now

Reply:
good catch